Content.Server/GameTicking/GameTicker.cs

```diff
@@ -531,15 +534,15 @@ public bool PauseStart(bool pause = true)
 
             if (pause)
             {
-                _pauseTime = DateTime.UtcNow;
+                _pauseTime = IoCManager.Resolve<IGameTiming>();
```

Comment:
No. Use the dependency you have, and get `RealTime` instead of setting `_pauseTime` to `IGameTiming`.